Source/MQTTnet/Internal/AsyncQueue.cs

```diff
@@ -32,6 +36,16 @@ public void Dispose()
                 _signal.Dispose();
 
                 _isDisposed = true;
+
+#if !NETSTANDARD1_3
+                if (typeof(IDisposable).IsAssignableFrom(typeof(TItem)))
```

Comment:
For the use within MQTTnet not needed, as no `TItem` implements `IDisposable`, but this type is `public` so it should dispose of all items in the queue.

For NS1.3 there's no easy way to perform that check upfront, only within the loop, so I skipped it for that target. It wasn't done before, so there's no regression for NS1.3.

Reply:
This is nor related to pull but to point about NS.... Why is NS1.3 even still supported isnt better to drop that support in next major release at all ? I know there can be legacy but still let have a look on what is TLS and supported by ofiicial microsoft releases...

Reply:
@damikun  This goal of this library is that it runs on most .NET versions. Especially because it covers a hardware topic. There are quite old machine out there which may run very old frameworks like 4.5.2 on XP machines. They cannot easily upgrade because they run on very special hardware. I also know people which use the library on a Pi 2 with an old version of UWP. I also wanted to support .NET 3.5 (which has support until 2029) but I requires too much changes.